README.md

```diff
@@ -754,7 +1023,28 @@ Note that custom block filter rules can also be created in code via [the `vip_bl
 
 ## Code Filters
 
-Block Data API filters can be applied to limit access to the REST API and modify the output of parsed blocks.
+### GraphQL
+
+```php
+/**
+	* Filter to enable/disable the graphQL API. By default, it is enabled.
+	* 
+	* @param bool $is_graphql_to_be_enabled Whether the graphQL API should be enabled or not.
+*/
+apply_filters( 'vip_block_data_api__is_graphql_enabled', true );
+```
+
```

Comment:
It's a little odd to include the `apply_filters` definition here, which isn't relevant to usage. I would remove it

```suggestion
```